src/main/daml/Daml/Finance/Interface/Lifecycle/Rule/Claim.daml

```diff
@@ -29,11 +29,11 @@ data ClaimResult = ClaimResult
   deriving (Eq, Show)
 
 -- | Interface for contracts that allow holders to claim an `Effect` and generate `SettlementInstruction`\s.
-interface Settlement where
+interface Claim where
   viewtype V
 
-  claim : Claim -> Update ClaimResult
-    -- ^ Implementation of the `Claim` choice.
+  effectuate : Effectuate -> Update ClaimResult
```

Comment:
Is it too ugly if we call the choice `ClaimEffect`? I find it quite descriptive.

Reply:
I changed to that. Agree, that it is more descriptive.